app/models/image.rb

```diff
@@ -4,6 +4,7 @@ class Image < ActiveFedora::Base
   include ::Hyrax::WorkBehavior
   include ::OregonDigital::TriplePoweredProperties::WorkBehavior
   include ::OregonDigital::ImageMetadata
+  include ::OregonDigital::GenericMetadata
```

Comment:
included at the bottom of the file, which one should remain..remove one or the other